docs/src/pages/docs/functions/pointfree-functions.md

```diff
@@ -75,6 +75,7 @@ accepted Datatype):
 | `merge` | `(a -> b -> c) -> m a b -> c` | `crocks/pointfree` |
 | [`nmap`][nmap] | `Integer -> ...(* -> *) m ...* -> m ...*` | `crocks/Tuple` |
 | `option` | `a -> m a -> a` | `crocks/pointfree` |
+| `project` | `Integer -> m ...* -> a` | `crocks/pointfree` |
```

Comment:
This need to point to a documentation section on the `Tuple` crocks page.
See `nmap` for an example

Reply:
Also needs an entry in the table down below